src/test/scala/it/pagopa/interop/authorizationmanagement/model/persistence/impl/ClientManagementSpec.scala

```diff
@@ -37,7 +37,6 @@ class ClientManagementSpec
 
     "be created successfully" in {
```

Comment:
can you add a test where the `clientId` is not passed, as the previous version, to verify backward compatibility?